Jenkinsfile

```diff
@@ -30,7 +30,9 @@ node('psi_rhel8') {
           credentialsId: 'browserstack',
           usernameVariable: 'BROWSERSTACK_USER',
           passwordVariable: 'BROWSERSTACK_KEY'
-        )
+        ),
+        string(credentialsId: 'firebase-server-key', variable: 'FIREBASE_SERVER_KEY'),
+        string(credentialsId: 'firebase-sender-id', variable: 'FIREBASE_SENDER_ID'),
```

Comment:
Should we update readme with these variables as a prerequisite when someone is running the tests locally?

Reply:
yes definetly, maybe it is also good to verify that these variables are set before starting the tests